src/func/grammar.ts

```diff
@@ -1407,23 +1418,33 @@ export type FuncAstComment = FuncAstCommentSingleLine | FuncAstCommentMultiLine;
 export type FuncAstCommentSingleLine = {
     kind: "comment_singleline";
     line: string;
+    style: ";" | ";;";
```

Comment:
@jubnzv Single `;` is not a comment, but an empty statement! Single-comments must start with `;;`

Reply:
Thanks! That's right